pages/create-team.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { connect } from 'unistore/react'
+import { actions } from '../lib/store'
+import TeamDetailsForm from '../components/teams/TeamDetailsForm'
+import Router from '../lib/router'
+
+function CreateTeam (props) {
+  const handleSubmit = async (data) => {
+    try {
+      await props.createTeam(data)
+      Router.push('/teams')
+    } catch (e) {
+      console.error(e)
```

Comment:
Consider removing this try/catch block. Unhandled exceptions should get logged to console anyways.

At first it looks like this block could fail silently for the user if there was an exception. But I believe `createTeam` prop is coming from `lib/store/actions/teams.js`, which has state management/notification and logging of the exceptions in it's `createTeam()`. If that is indeed case, then the try/catch block here is not doing anything and could be removed entirely.

Reply:
I think the idea here was to make sure that a Router push doesn't happen when there is an error (copied pattern from another part of the codebase). I'm going to take a look at the store and make sure that an exception doesn't still trigger the Router code.

Reply:
Line 11 should not be reached at all if there is an exception from line 10- regardless of the try/catch block being there.

Reply:
@kamicut can you respond or edit on this one?

Reply:
I removed the try catch @guidorice